sdk/avs/Microsoft.Azure.Management.Avs/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I didn't realize this didn't have an assembly info.  It should have all the appropriate properties in here.  Here is a sample: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

Assembly Version should be 1.0.0, Metadata should match this assembly name.

Reply:
I added the additional assembly info metadata as requested in 19b8c13. I set the `AssemblyVersion` to `1.0.0` as well.